src/query/functions/temporal/aggregation_test.go

```diff
@@ -128,6 +137,19 @@ var testCases = []testCase{
 			{2, 2, 2, 2, 2},
 		},
 	},
+	{
+		name:   "holt_winters",
+		opType: HoltWintersType,
```

Comment:
Since holt winters is a fair bit more complex than some of the other agg functions, might be a good idea to have a test or two that hit the function directly rather than building the blocks etc?